### PR TITLE
add version info in panel

### PR DESCRIPTION
**Pull request by PeterDaveHello**
So we don't need to open store to know the current version!

### CODE REVIEW

**Comment by arthurvr** [comment]
Isn't the store simply a better place to keep track of this? That's where everybody will logically be searching, and we don't have to update this version every time.

**Comment by PeterDaveHello** [comment]
hmmmm ... I think this can be local, so no need to open the website, just like this project does - get notifications but don't need to open the website, faster and easier if I just wanna know the version. But this is just my point :smile:

File: extension/options.html [hunk 1]
```diff
@@ -23,6 +23,10 @@
 			#success_message.visible {
 				opacity: 1;
 			}
+			
+			#id {
```
Comment (attilaolah) [inline extension/options.html]: The `id` of the `<p>` is `ver`, not `id`.

**Comment by attilaolah** [comment]
> and we don't have to update this version every time

:+1: 

Plus, I don't think knowing the version should be that important to the end user.

Plus, you don't really need to go to the store to see the version, just visit chrome://extensions/ or navigate from the menu.

**Comment by PeterDaveHello** [comment]
Oh yeah, that's true!

**Comment by arthurvr** [comment]
Agreed.

**Pull request closed without merging**